packages/assistify-ai/server/SmartiRouter.js

```diff
@@ -27,7 +27,8 @@ RocketChat.API.v1.addRoute('smarti.result/:_token', {authRequired: false}, {
 		if (this.urlParams._token && this.urlParams._token === rcWebhookToken) {
 
 			SystemLogger.debug('Smarti - got conversation result:', JSON.stringify(this.bodyParams, null, 2));
-			SmartiAdapter.analysisCompleted(this.bodyParams.channelId, this.bodyParams.conversationId, this.bodyParams.token);
+			SmartiAdapter.analysisCompleted(this.bodyParams.channelId, this.bodyParams.conversationId, Date.now(), this.bodyParams.token);
```

Comment:
@ruKurz after y'day's decision to use the timestamp of the smarti conversation for the mapping information, why do we pass `Date.now()` into this method actually?